CP04OSSM/CP04OSSM_R00002_ingest.csv

```diff
@@ -1,43 +1,43 @@
+#mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/syslog/*.syslog.log,CP04OSSM-RID27-00-DCLENG000,recovered_host,Not Expected,no recovered ENG files
+mi.dataset.driver.optaa_dj.dcl.optaa_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/OPTAA_snACS240/*.optaa1.log,CP04OSSM-RID27-01-OPTAAD000,recovered_host,Available,
+mi.dataset.driver.flort_dj.dcl.flort_dj_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/FLORT_sn994/*.flort.log,CP04OSSM-RID27-02-FLORTD000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.dcl.ctdbp_cdef_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/CTDBP_sn16P71174-7208/*.ctdbp1.log,CP04OSSM-RID27-03-CTDBPC000,recovered_host,Available,
+mi.dataset.driver.ctdbp_cdef.ctdbp_cdef_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/CTDBP_sn16P71174-7208/SBE*.hex,CP04OSSM-RID27-03-CTDBPC000,recovered_inst,Available,
+mi.dataset.driver.dosta_abcdjm.dcl.dosta_abcdjm_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl27/DOSTA_sn410/*dosta1.log,CP04OSSM-RID27-04-DOSTAD000,recovered_host,Available,
+#mi.dataset.driver.cg_cpm_eng.cpm.cg_cpm_eng_cpm_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/irid2shore/cpm_status*.txt,CP04OSSM-SBC11-00-CPMENG000,recovered_host,Not Expected,no recovered ENG files
+#mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/syslog/*.syslog.log,CP04OSSM-SBD11-00-DCLENG000,recovered_host,Not Expected,no recovered ENG files
+mi.dataset.driver.cg_stc_eng.stc.mopak_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/MOPAK_sn/*.mopak.log,CP04OSSM-SBD11-01-MOPAK0000,recovered_host,Available,
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/Hydrogen_sn/*.hyd1.log,CP04OSSM-SBD11-02-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.metbk_a.dcl.metbk_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl11/METBK-Log_sn/*.metbk.log,CP04OSSM-SBD11-06-METBKA000,recovered_host,Available,
+#mi.dataset.driver.cg_dcl_eng.dcl.cg_dcl_eng_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl12/syslog/*.syslog.log,CP04OSSM-SBD12-00-DCLENG000,recovered_host,Not Expected,no recovered ENG files
+mi.dataset.driver.hyd_o.dcl.hyd_o_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl12/Hydrogen_sn/*.hyd2.log,CP04OSSM-SBD12-03-HYDGN0000,recovered_host,Available,
+mi.dataset.driver.pco2a_a.dcl.pco2a_a_dcl_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl12/PCO2A_sn34-229-50A/*.pco2a.log,CP04OSSM-SBD12-04-PCO2AA000,recovered_host,Available,
+#mi.dataset.driver.zplsc_c.zplsc_c_recovered_driver,/omc_data/whoi/OMC/CP04OSSM/R00002/cg_data/dcl37/ZPLSC_sn55077/DATA/*/*.01A,CP04OSSM-MFD37-07-ZPLSCC000,recovered_inst,Available,Test parser
```

Comment:
Commented out this line because we cannot correctly ingest ZPLSC recovered instrument data yet

Reply:
Recommend moving this up to the other ZPLSC line.
The MFN went down the day after deployment, so there should be no (or 1 day) of recovered-host data for MFN instruments.  All MFN recovered host lines should be commented out.
(ADCP PD8 data should not be ingested anyway).